Core/include/Acts/Definitions/Common.hpp

```diff
@@ -36,6 +36,11 @@ static constexpr ActsScalar s_curvilinearProjTolerance = 0.999995;
 /// respect to a given momentum or direction
 enum class NavigationDirection : int { Backward = -1, Forward = 1 };
 
+/// Convert navigation dir to index
```

Comment:
Maybe add some documentation what this index refers to? since at least for me this is not immediately clear...

Reply:
Fixed.